_pages/labwiki.md

```diff
@@ -8,23 +8,251 @@ permalink: /labwiki/
+- Work on what you’re passionate about, work hard at it, and be proud of it. Be so proud of it that you have to suppress bragging (but it’s ok to brag sometimes).
+- Scientists have to be careful. Don’t rush your work. Think about it. Implement it. Double and triple check it. Incorporate sanity checks. Ask others to look at your code or data if you need help or something looks off. It’s ok to makes mistakes, but mistakes shouldn’t be because of carelessness or rushed work.
+- If you do make a mistake, you should definitely tell your collaborators (if they have already seen the results, and especially if the paper is being written up, is already submitted, or already accepted). We admit our mistakes, and then we correct them and move on. We all want to get papers published and do great things. But we do this honestly. It is never ok to plagiarize, tamper with data, make up data, omit data, or fudge results in any way. Science is about finding out the truth, and null results and unexpected results are still important. This can’t be emphasized enough: no academic misconduct!
+- Support your fellow lab-mates. Help them out if they need help (even if you aren’t on the project), and let them vent when they need to. Science is collaborative, not competitive. Help others, and you can expect others to help you when you need it.
+- Respect your fellow lab-mates. Respect their strengths and weaknesses, respect their desire for quiet if they need it, and for support and a kind ear when they need that. Respect their culture, their religion, their beliefs, their sexual orientation.
+- If you’re struggling, tell someone (feel free to tell JB!). Your health and happiness come first. The lab looks out for the well-being of all its members. We are here to help. It’s ok to go through hard patches (we all do), but you shouldn’t feel shy about asking for help or just venting.
+- If there is any tension or hostility in the lab, something has to be done about it immediately. We can’t thrive in an environment we aren’t comfortable in, and disrespect or rudeness will not be tolerated in the lab. If you don’t feel comfortable confronting the person in question, tell JB. In any case, tell JB.
+- If you have a problem with JB and are comfortable telling him about it, do! If you aren’t comfortable, then tell your IPN mentor (if you are an IPN student), or else follow [these guidelines](https://www.mcgill.ca/hr/self-service/resolve-workplace-conflict) for resolving workplace conflicts at McGill.
+- Stay up to date on the latest research, by using RSS feeds and/or getting journal table of contents. Also consider following scientists in the field on Twitter
+- Remember the lab philosophy: **[JB, do you have a lab philosophy?]**. Have a life outside of the lab, take care of your mental and physical health, and don’t ever feel bad for taking time off work.
```

Comment:
That lab philosphy sounds right to me - I would add : take care of each others as well, strive to do some good work for the community

Reply:
@jbpoline sounds good! In manual I was copying, this bullet point was this:
- Remember the lab philosophy: “We like to do good science and have fun. At the same
time, but also separately”. Have a life outside of the lab, take care of your mental and
physical health, and don’t ever feel bad for taking time off work.

If you don't have a specific lab philosophy quote, maybe we could change it to this:
- Have a life outside of the lab, take care of your mental and physical health, and don’t ever feel bad for taking time off work. Take care of each others as well, and strive to do some good work for the community.

What do you think?

Reply:
@koudyk  : Sounds good - let's add the "fun" to it, so: 
"Take care of each others as well, strive to do some good work for the community, have fun in the process".